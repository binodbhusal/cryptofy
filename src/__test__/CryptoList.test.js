import { Provider } from 'react-redux';
import { MemoryRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import Cryptolist from '../__MockTest__/CryptoList';
import store from '../__MockTest__/mockstore';

describe('CryptoList component render', () => {
  test('Test render of Cryptolist component', () => {
    const { container } = render(
      <MemoryRouter>
        <Provider store={store}>
          <Cryptolist />
        </Provider>
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
