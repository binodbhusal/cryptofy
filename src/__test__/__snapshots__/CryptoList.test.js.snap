// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CryptoList component render Test render of Cryptolist component 1`] = `
<div>
  <p>
    Loading...
  </p>
</div>
`;
